src/NumericalAlgorithms/LinearAlgebra/CMakeLists.txt

```diff
@@ -3,9 +3,7 @@
 
 set(LIBRARY LinearAlgebra)
 
-set(LIBRARY_SOURCES
-    FindGeneralizedEigenvalues.cpp
-   )
+set(LIBRARY_SOURCES FindGeneralizedEigenvalues.cpp)
```

Comment:
Sorry, what we meant was to have two spaces like what is used with target_link_libraries below. Have the list on different lines is good because that way if multiple people add to the file the conflicts will be avoided or minimal :) Sorry, for not being clearer before